buildSrc/src/main/java/com/yelp/gradle/bento/GlobalDependencies.kt

```diff
@@ -5,31 +5,31 @@ import org.gradle.api.artifacts.repositories.MavenArtifactRepository
 
 object Publishing {
     const val GROUP = "com.yelp.android"
-    const val VERSION = "18.2.1"
+    const val VERSION = "19.0.0-SNAPSHOT"
 }
 
 object Versions {
-    const val COMPILE_SDK = 28
-    const val TARGET_SDK = 28
+    const val COMPILE_SDK = 31
+    const val TARGET_SDK = 31
     const val MIN_SDK = 21
 
     // In alphabetical order.
     const val APACHE_COMMONS = "3.4"
-    const val ANDROID_GRADLE = "4.2.0"
+    const val ANDROID_GRADLE = "7.1.2"
     const val ANDROID_X_APP_COMPAT = "1.0.0"
     const val ANDROID_X_CONSTRAINT_LAYOUT = "1.1.2"
     const val ANDROID_X_CORE_CTX = "1.4.0"
     const val ANDROID_X_LIFECYCLE = "2.3.1"
     const val ANDROID_X_MATERIAL = "1.0.0"
     const val ANDROID_X_RECYCLER_VIEW = "1.0.0"
     const val ANDROID_X_TEST = "1.1.0"
+    const val COMPOSE = "1.1.1"
     const val COROUTINES = "1.4.0"
     const val ESPRESSO = "3.1.0"
-    const val GRADLE = "6.9"
+    const val GRADLE = "7.4.1"
     const val GUAVA = "28.1-android"
     const val JUNIT = "4.12"
-    const val KOTLIN = "1.3.72"
-    const val MAVEN_PUBLISH = "3.6.2"
+    const val KOTLIN = "1.6.10"
```

Comment:
latest is now 1.6.20
main repo has 1.6.20 as well

Reply:
I won't bump that because compose isn't compatible.